base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/commands/DeleteComponentsHandler.java

```diff
@@ -48,6 +49,16 @@
 	public DeleteComponentsHandler() {
 		super(SERVER.deleteComponents());
 	}
+	
+	/**
+	 * Opens dialog to confirm deleting components
+	 * @param selectedConfigs Components that will be displayed in confirmation
+	 * @return
+	 */
+    private boolean deleteComponentsConfirmDialog(Collection<String> selectedComponents) {
+        return MessageDialog.openQuestion(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell(), "Confirm Delete Components",
+                "The following components " + selectedComponents + " will be permanently deleted. Are you sure you want to delete them?");
+    }
 
```

Comment:
Wondering if there would be a way to generalise this for components AND configs, maybe we should pull it out into a helper file if possible as there's some duplication?